cfgov/jinja2/v1/sublanding-page/index.html

```diff
@@ -62,26 +50,22 @@
 
 {% block content_sidebar scoped -%}
     {% for block in page.sidefoot %}
-        {% if 'related_posts' in block.block_type %}
-            {{ related_posts.render(block) }}
-        {% elif 'email_signup' in block.block_type %}
-            {{ render_stream_child(block) }}
-        {% else %}
-            {% if 'heading' in block.block_type %}
+        <div class="block block__flush-top">
```

Comment:
@richaagarwal i noticed all our sidefoot blocks in other templates added this div class `<div class="block block__flush-top">` to resolve spacing issues. noticed it was missing here, so included it in